### PR TITLE
[kommander] Porting the workspace catalog structure to 2.2

### PR DESCRIPTION
**Pull request by cbuto**
## Description of changes being made

This is porting the work done in https://github.com/mesosphere/dcos-docs-site/pull/3952 and https://github.com/mesosphere/dcos-docs-site/pull/3955/files to the 2.2 docs. This PR diff should look the same as #3952 and https://github.com/mesosphere/dcos-docs-site/pull/3955/files

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](./CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/workspaces/applications/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Applications
+title: Applications
+menuWeight: 30
+excerpt: Applications
+---
+This section documents the applications and application types that you can utilize with Kommander.
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/workspaces/applications/index.md:9]:
```suggestion
This section documents the applications and application types that you can use with Kommander.
```

Comment (cbuto) [reply pages/dkp/kommander/2.2/workspaces/applications/index.md]: Fixed in both 2.1 and 2.2 (ddc6dea9b5fa3fc04e28ebdc2a608fda6b62bcc8)

File: pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: layout.pug
+navigationTitle: Workspace Catalog Applications
+title: Workspace Catalog Applications
+menuWeight: 20
+excerpt: Workspace Catalog Applications
+---
+<!-- TODO: settle down the link target to custom applications -->
+Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
```
[the remark this answers is not in the document]
Comment (cbuto) [reply pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/index.md]: Fixed in both 2.1 and 2.2 (ddc6dea9b5fa3fc04e28ebdc2a608fda6b62bcc8)!

**Comment by cbuto** [comment]
I updated this PR with the other changes made in https://github.com/mesosphere/dcos-docs-site/pull/3955/files. Going forward I wonder if we should handle 2.1 and 2.2 in a single PR to avoid reviewing the same content twice.

**Review by cwyl02 — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Comment by cbuto** [comment]
@cmastr do you mind taking one more look at this? There shouldn't be any new content here, just copying everything over to 2.2!

File: pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ This takes you to the **Applications** page which displays all applications that
 
 To use the CLI to deploy or uninstall applications, see [Application Deployment](./application-deployment)
 
-<p class="message--important"><strong>IMPORTANT: </strong>There may be dependencies between the applications, which are listed <a href="./platform-service-dependencies/">here</a>. Review them carefully prior to customizing to ensure that the applications are deployed successfully.</p>
+<p class="message--important"><strong>IMPORTANT: </strong>There may be dependencies between the applications, which are listed <a href="./platform-service-dependencies/">in this documentation</a>. Review them carefully prior to customizing to ensure that the applications are deployed successfully.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md:28]:
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong>There may be dependencies between the applications, which are listed <a href="./platform-service-dependencies/">in the platform service dependencies</a> documentation. Review them carefully prior to customizing to ensure that the applications are deployed successfully.</p>
```

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
> @cmastr do you mind taking one more look at this? There shouldn't be any new content here, just copying everything over to 2.2!

Added one suggestion, but overall this looks good. There are a few Vale things, but no blockers IMO.

File: pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md (hunk 3 above)
Comment (cbuto) [reply pages/dkp/kommander/2.2/workspaces/applications/platform-applications/index.md]: @cmastr fixed in c1f9e334d9beac07887067b06f19fc362dbbec8a!

**Pull request merged**